apps/user_profile/models.py

```diff
@@ -2,30 +2,32 @@
 from django.contrib.auth.models import User
 from django.db import models
 
+from apps.location.models import City
 from core.models.helpers import image_path
 
 
 class UserProfile(models.Model):
     SEX_CHOICES = (
-        ('F', 'female'),
-        ('M', 'male')
+        ('Жіноча', 'female'),
```

Comment:
its true, why did you change it?

Reply:
In Django templates you can use the "get_FOO_display()" method, that will return the readable alias for the field, where 'FOO' is the name of the field.